latest/index.bs

```diff
@@ -230,37 +228,895 @@ Metadata {#metadata}
+        the provided `path` MUST exist in this container.</dd>
+  <dt><strong>interpolation</strong></dt>
+  <dd>  The `interpolation` attributes MAY be provided. It's value indicates
+        the interpolation to use if transforing points not on the array's discrete grid.
+        Values could be:
+        <ul>
+            <li><code>linear</code> (default)</li>
+            <li><code>nearest</code></li>
+            <li><code>cubic</code></li>
+        </ul></dd>
+</dl>
+
+The array data at `path` MUST define space and coordinate transform metadata:
+
+* space metadata MUST have exactly one axis with `"type" : "coordinate"`
+* Every axis name in the `coordinateTransform`'s `input` MUST appear in the space
+* The array dimension corresponding to the coordinate axis MUST have length greater than or equal to the dimensionality of the `coordinateTransform` `output`
+* SHOULD have `name` equal to the `name` of the corresponding `coordinateTransform`.
+
+For `coordinates`:
+
+* If a `coordinates`' input space has dimensionality `N`, then the array data at `path` MUST have dimensionality equal to `(ni + 1)`.
+* space metadata MUST have exactly one axis with `"type" : "coordinate"`
+
+For `displacements`:
+* space metadata MUST have exactly one axis with `"type" : "displacement"`
+* `input` and `output` MUST have an equal number of dimensions.
+
+For example, in 1D:
+```
+{
+    "name" : "a coordinate field transform",
+    "type": "coordinates",
+    "path" : "i2xCoordinates",
+    "input" : ["i"],
+    "output" : ["x"],
+    "interpolation" : "nearest"
+}
+```
+
+if the arrray in `coordinates` contains the data: `[-9, 9, 0]`, then this metadata defines the function:
+
+```
+x = 
+    if ( i < 0.5 ) -9
+    else if ( i < 1.5 ) 9
+    else 0
+```
+
+Example metadata for the array data at path `coordinates` above:
+
+```
+{
+    "spaces" : [
+        {
+            "name" : "a coordinate field transform",
+            "axes" : [
+                { "label": "i", "type": "space", "discrete": true },
+                { "label": "c", "type": "coordinate", "discrete": true }
+            ]
+        } 
+    ],
+    "transformations" : [
+        {
+            "type" : "identity",
+            "output" : "a coordinate field transform"
+        }
+    ]
+}
+```
+
+A 1D example displacement field:
+```
+{
+    "name" : "a displacement field transform",
+    "type": "displacements",
+    "path" : "displacements",
+    "input" : ["x"],
+    "output" : ["y"],
+    "interpolation" : "linear"
+}
+```
+
+if the arrray in `displacements` contains the data: `[-1, 0, 1]`.
+
+Example metadata for the array data at path `displacements` above:
+
+```
+{
+    "spaces" : [
+        {
+            "name" : "a coordinate field transform",
+            "axes" : [
+                { "label": "x", "type": "space", "unit" : "nanometer" },
+                { "label": "d", "type": "displacement", "discrete": true }
+            ]
+        } 
+    ],
+    "transformations" : [
```

Comment:
This doesn't look like the right specification. Should this be more like:

```suggestion
    "coordinateSystems" : [
        {
            "name" : "a coordinate field transform",
            "axes" : [
                { "name": "x", "type": "space", "unit" : "nanometer" },
                { "name": "d", "type": "displacement", "discrete": true }
            ]
        } 
    ],
    "coordinateTransformations" : [
```

Reply:
thanks - fixed!